spec/presenters/hyrax/collection_presenter_spec.rb

```diff
@@ -50,19 +50,10 @@
 
       it 'finds the collection type based on the solr_document#collection_type_gid if one exists' do
         expect(solr_doc).to be_key('collection_type_gid_ssim')
-        expect(solr_doc).to receive(:fetch).with('collection_type_gid_ssim', Array).and_return(collection_type.gid)
+        expect(solr_doc['collection_type_gid_ssim']).to eq [collection_type.gid]
```

Comment:
I would just remove this assertion (and the previous line). It's not testing the code in `CollectionPresenter`, which is what this test should be about